gradle/libs.versions.toml

```diff
@@ -98,6 +99,7 @@ commons-text = { module = "org.apache.commons:commons-text", version.ref = "comm
 compose-accompanist-permissions = { module = "com.google.accompanist:accompanist-permissions", version.ref = "compose-accompanist" }
 compose-bom = { group = "androidx.compose", name = "compose-bom", version.ref = "compose-bom" }
 compose-material = { module = "androidx.compose.material:material" }
+compose-material3 = { group = "androidx.compose.material3", name = "material3-android" }
```

Comment:
Where have you gotten that dependency from? According to [the documentation](https://developer.android.com/jetpack/androidx/releases/compose-material3) it should be `androidx.compose.material3:material3` right?

Reply:
Good point. I have added the new M3 `AppTheme` in Android Studio before the dependency. Android Studio then suggests:

![Bildschirmfoto vom 2024-04-15 19-14-40](https://github.com/bitfireAT/davx5-ose/assets/156167/8edc8572-a9e9-4aa6-9a3a-62047864e597)

→

![Bildschirmfoto vom 2024-04-15 19-14-49](https://github.com/bitfireAT/davx5-ose/assets/156167/61a811f2-ad36-4039-966b-a93804cadcc3)

🤔

Reply:
I guess it's because of Compose Multiplatform? 🤔 They can also be just the same thing with different names. I don't know

Reply:
Maybe the material3 dependency internally includes material3-android or -desktop, depending on the actual platform. Android Studio maybe then has just suggested the actual library where the thing is defined and not the "meta" library. 

Who knows... surely better to include it like the docs say :)

Reply:
> Who knows... surely better to include it like the docs say :)

Yeah 😅